.gitignore

```diff
@@ -26,7 +26,7 @@
 /tmp/
 
 # Used by dotenv library to load environment variables.
-# .env
+.env
 
```

Comment:
Not sure If you've had to deal with this, but it's a pain if you accidentally commit something to github that should be secret. So, nice job!